### ISSUE TITLE
intermitten error

### ISSUE THREAD

**Issue by lawa42** [westandskif/rate-arch-mirrors#3]
Hi

I am using rate-arch-mirrors via the suggested alias. However, from time to time I get the following error message:

```thread 'main' panicked at '`at` split index (is 5) should be <= len (is 0)'```

The suggest backtracing is difficult as the tool always works the second time.

The full output looks like this
```# JUMP #1
# EXPLORING DE
# VISITED DE
#     + NEIGHBOR US (by HubsFirst)
#     + NEIGHBOR GB (by HubsFirst)
#     + NEIGHBOR CN (by HubsFirst)
#     + NEIGHBOR FR (by DistanceFirst)
#     + NEIGHBOR PL (by DistanceFirst)
#     + NEIGHBOR DK (by DistanceFirst)
thread 'main' panicked at '`at` split index (is 5) should be <= len (is 0)', library/alloc/src/vec/mod.rs:1751:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
# blank iteration

# RE-TESTING TOP MIRRORS
==> no candidate packages found for pruning
:: Synchronizing package databases...
error: failed to update core (no servers configured for repository)
error: failed to update extra (no servers configured for repository)
error: failed to update community (no servers configured for repository)
error: failed to update multilib (no servers configured for repository)
error: failed to synchronize all databases
Command 'sudo pacman --color=always --sync --refresh --refresh' failed to execute.
Do you want to retry? [Y/n] n
Do you want to proceed? [y/N] n
```

**Comment by westandskif** [comment]
Hi @lawa42 , thank you!
This should be fixed in [v0.3.0](https://github.com/westandskif/rate-arch-mirrors/blob/master/CHANGELOG.md)

**Comment by lawa42** [comment]
upgraded and ran it a few times no errors so far. thanks
